<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="readButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>load gestures</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>540</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>save gestures</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>191</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="listButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>list gestures</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>650</width>
      <height>490</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>840</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>readButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>read()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadGestures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captureButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>capture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveGestures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>677</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>listGestures()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>read()</slot>
  <slot>loadGestures()</slot>
  <slot>saveGestures()</slot>
  <slot>capture()</slot>
  <slot>listGestures()</slot>
 </slots>
</ui>
